nomos-utils/Cargo.toml

```diff
@@ -1,21 +1,21 @@
 [package]
 edition = "2021"
 license = { workspace = true }
-name = "nomos-utils"
+name    = "nomos-utils"
 version = "0.1.0"
 
 [dependencies]
-const-hex = "1"
-humantime = { version = "2.1", optional = true }
-rand = "0.8"
+const-hex   = "1"
+humantime   = { version = "2.1", optional = true }
+rand        = "0.8"
 rand_chacha = "0.3"
-serde = { version = "1.0", optional = true, features = ["derive"] }
-serde_with = { version = "3.12", optional = true }
-time = { version = "0.3", optional = true, features = ["parsing", "serde"] }
+serde       = { version = "1.0", optional = true, features = ["derive"] }
+serde_with  = { version = "3.12", optional = true }
+time        = { version = "0.3", optional = true, features = ["serde-human-readable"] }
```

Comment:
Ah, this is a good catch actually!
This would affect #1156 I think

Reply:
Should we be using this in all `time` imports?